examples/defaulter-gen/generators/defaulter.go

```diff
@@ -562,8 +558,20 @@ func populateDefaultValue(node *callNode, t *types.Type, tags string, commentLin
 		node.markerOnly = true
 	}
 
-	node.defaultIsPrimitive = t.IsPrimitive()
```

Comment:
Unrelated diff? Or is `node` or `t` being effected by the added codepath?

Reply:
This line was moved to line 574. The diff just renders as this line being removed :/